app/build.gradle.kts

```diff
@@ -32,8 +32,8 @@ android {
         }
     }
 
-    signingConfigs {
-        create("release") {
+    val releaseSigning = try {
+        signingConfigs.create("release") {
```

Comment:
Update here due to https://github.com/Goooler/Noto/actions/runs/9399185433/job/25886111356.